packages/beacon-node/test/unit/eth1/jwt.test.ts

```diff
@@ -10,6 +10,14 @@ describe("ExecutionEngine / jwt", () => {
     expect(decoded).toEqual(claim);
   });
 
+  it("encode/decode correctly with id and clv", () => {
+    const jwtSecret = Buffer.from(Array.from({length: 32}, () => Math.round(Math.random() * 255)));
+    const claim = {iat: Math.floor(new Date().getTime() / 1000), id: "4ac0", clv: "v1.11.3"};
```

Comment:
As tests can serve as documentation of code behavior a lot of time, could consider using a proper clv value (as Lodestar would include it)